docs/ci-integration/guides/bitbucket-pipelines-integration.md

```diff
@@ -0,0 +1,40 @@
+# Bitbucket Pipelines integration
+
+Bitbucket Pipelines run in a shared Docker environment and do not support running Earthly builds directly due to [restrictions](https://jira.atlassian.com/browse/BCLOUD-21419) that Bitbucket has put in place.
+
+You can however, run Earthly builds on Bitbucket pipelines via [Earthly Satellites](../../cloud/satellites.md). Because Bitbucket Pipelines run as containers you can also use the official Earthly Docker image. Here is an example of a Bitbucket Pipeline build. This example assumes `+unit-test` and `+build` targets in your Earthfile.
```

Comment:
I would point toward [remote runners](https://docs.earthly.dev/docs/remote-runners) to make it a bit more OSS-friendly. We could say "remote runners, such as Earthly Satellites" to link both.

Reply:
Done